src/assets/CollectiblesController.ts

```diff
@@ -803,6 +816,15 @@ export class CollectiblesController extends BaseController<
 
   private getERC1155TokenURI: AssetsContractController['getERC1155TokenURI'];
 
+  private onCollectibleAdded: (sensitiveProperties: {
+    token_contract_address: string;
+    token_symbol?: string;
```

Comment:
[nit] It doesn't seem like `token_symbol` is ever left out of the object, so would a more appropriate type be `token_symbol: string | undefined`? This is especially relevant now that TypeScript 4.4 has the ability to treat optional properties and undefined properties as separate concepts.